File: #03/Categoria 04/q04 - elemento/elemento.js
elemento = (n, arr) => {
    if (n < 0) return arr[n + arr.length]
    else return arr[n]
}

console.log(elemento(2, [2,7,3,9]));
console.log(elemento(0, [2,7,3,9]));
console.log(elemento(-1 , [2,7,3,9]));
console.log(elemento(-2, [2,7,3,9]));

File: #03/Categoria 04/q04 - elemento/elemento.hs
elemento :: Int -> [a] -> a
elemento n xs =
    if n < 0
     then let p = n + length xs in xs !! p
    else xs !! n





main :: IO ()
main = do
    a <- readLn :: IO Int
    b <- readLn :: IO [Int]
    print $ elemento a b